e2e/runTest.sh

```diff
@@ -36,11 +36,16 @@ fi
 bsp_path="$(bazel info bazel-bin)/server/src/main/java/org/jetbrains/bsp/bazel/bsp-install"
 echo "Building done."
 
-echo "Installing BSP..."
+echo "Cleaning project directory..."
 if [ "$#" -eq 2 ]; then
   cd "$TEST_PROJECT_PATH" || exit
 fi
-$bsp_path
+rm -r .bsp/ > /dev/null 2>&1
```

Comment:
The `rm` statements still look a little scary, but I don't speak a lot of `sh`. What happens if `[ "$#" -eq 2 ]` is not met? Maybe move the `rm` statements into the `then` scope.

Reply:
by default test is run in the same directory, if statement is only for the case where we want to run test somewhere else